### DISCUSSION TITLE
Corrupt downloads from Egnyte SFTP server

### DISCUSSION THREAD

**Discussion by furkantektas** [iterate-ch/cyberduck#15685] [Q&A] [answered]
We encountered file corruption on multiple files from Egnyte SFTP server. We can reproduce the same problem consistently on various file sizes ranging from 40-350MB. We haven't experienced any issues with different servers in the past.

The file contents and MD5 checksums are verified as correct when downloaded manually via [paramiko](https://www.paramiko.org/). This indicates the problem likely originates with Cyberduck.

**Desktop:**
 - OS: macOS 13.5.1 (22G90)
 - Version 8.6.3 (40040)

**~~Log Files~~**
_Unfortunately, due to customer security policies, I am unable to provide logs._

Without comprehensive logging and reproducible steps, I don't expect this issue can be definitively resolved.

I wanted to document this consistent problem as a reference for future issues. It seems very similar to #7641. I can help troubleshooting the error as it consistently corrupts the file in the same way. 

Thanks

**Comment by dkocher** [comment]
To narrow down the problem, can you reproduce when you disable [segmented downloads](https://docs.cyberduck.io/cyberduck/download/#segmented-downloads)?

**Comment by furkantektas** [comment, marked as answer]
Unfortunately, I am no longer working with the company, can't help reproducing the error. 

Back in the day, I fixed the issue by a custom downloader using [paramiko](https://www.paramiko.org). 

Please feel free to close as I can't verify any longer.

Thanks
